compose-backstack/src/main/java/com/zachklipp/compose/backstack/BackstackFrame.kt

```diff
@@ -0,0 +1,44 @@
+package com.zachklipp.compose.backstack
+
+import androidx.compose.runtime.Composable
+
+/**
+ * Models a frame in a Backstack, with a unique [key] to identify it,
+ * and a [Content] function to display it.
+ *
+ * Use [toBackstackModel] to associate any [List] of UI models with
+ * with [Composable] code that can display them, suitable for use
+ * with [Backstack].
+ */
+interface BackstackFrame<out K : Any> {
```

Comment:
Nit: I think you might notice better recomposition performance if you mark this interface as `@Stable`. But also very possibly not, since it's going to always be in a List which is not stable. Would be something to measure/test.